pages/docs/quickstart.mdx

```diff
@@ -84,7 +84,7 @@ To do this, you need a GitHub account. If you do not have one yet, you can creat
 | url   | string | Full url to the resource                                                                                                                                                        |
 | icon  | string | The icon you would like displayed. We support the following icon sets: **Fa** and **Si** from React Icons. You can search available [icons](https://linkfree.eddiehub.io/icons) |
 
-![propose new file](https://user-images.githubusercontent.com/82668196/208292185-743dc114-5699-4b58-bf5c-c5fc74f0248d.png)
+![propose new file](https://user-images.githubusercontent.com/94740455/227762486-daebf318-a814-486e-8b79-411dce878ea8.png)
```

Comment:
This doesn't show the fork information, which I think could be useful to people

Reply:
I Agree but the image before this shows the fork information quite clearly and I think this image should have more focus on the JSON code so that the user makes less mistakes.